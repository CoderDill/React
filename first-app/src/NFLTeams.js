const NFLTeams = [
  {
    name: "Miami Dolphins",
    stadium: "Hard Rock Stadium",
    capacity: 65326,
    img: "https://www.collegegridirons.com/wp-content/uploads/2017/03/hrs.jpg",
    logo: "https://lh3.googleusercontent.com/proxy/M5SrmZetHy66Cgm1yHVzkg0jE6zCs2nu8POex1BCa0cuAcWYwZ_xixF6lQyDUXARJokAVHeVV3Xzu9YPUPaCpcXTN3U0ynu9SHDNXzKd9qoOFE1_UI1odHliHPn5pgUntv-Vtc_bRg",
  },
  {
    name: "Arizona Cardinals",
    stadium: "Statefarm Stadium",
    capacity: 63400,
    img: "https://populous.com/wp-content/uploads/2018/02/POP_00_0796_00_N25_medium.jpg",
    logo: "https://lh3.googleusercontent.com/proxy/K_-WNZYkF6KiopB6FtZrdN6W5rP_4u7KBIoInwiAfPbnx_nR3KtwJ_T1rrr8c6-yagnBSlWst7ZdxHyeq4D5-qHvz5VsvSBJUH6ONecKCZMe1m0LDWtWw598sLAWTBR-hko",
  },
  {
    name: "Jacksonville Jaguars",
    stadium: "TIAA Bank Field",
    capacity: 67164,
    img: "https://downtownjacksonville.org/wp-content/uploads/2020/08/tiaa-bank-field.jpg",
    logo: "https://lh3.googleusercontent.com/proxy/NOhd8cahhw0xpf8vDi8vf0yNNm0EJuCRbEIxK3u8gz-00uz8BAFAm01Tbt_UAliQPrmx0VhmCO3lRoZnaVJVWMKp6Sf2tbc1WCW1plI59jDxLLo9VDrA0ptoPJBLaSueU9J1LQs",
  },
];

export default NFLTeams;
