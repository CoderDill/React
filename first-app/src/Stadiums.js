import React from "react";

const Stadiums = ({ NFLTeams }) => {
  return (
    <div>
      {NFLTeams.map((team) => (
        <div>
          <h2>{team.name}</h2>
          <img src={team.img} alt="" />
          <ul>
            <li>Stadium: {team.stadium}</li>
            <li>Capacity: {team.capacity}</li>
          </ul>
        </div>
      ))}
    </div>
  );
};

export default Stadiums;
