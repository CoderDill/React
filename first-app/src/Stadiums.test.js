import React from "react";
import { render } from "@testing-library/react";
import Stadiums from "./Stadiums";
import NFLTeams from "./NFLTeams";

test("it matches snapshot", () => {
  const { asFragment } = render(<Stadiums NFLTeams={NFLTeams}/>);
  expect(asFragment()).toMatchSnapshot();
});
