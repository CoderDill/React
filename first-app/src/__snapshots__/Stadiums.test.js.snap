// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="Logos"
    >
      <h2
        id="Miami Dolphins"
      >
        Miami Dolphins
      </h2>
      <img
        alt=""
        src="https://lh3.googleusercontent.com/proxy/M5SrmZetHy66Cgm1yHVzkg0jE6zCs2nu8POex1BCa0cuAcWYwZ_xixF6lQyDUXARJokAVHeVV3Xzu9YPUPaCpcXTN3U0ynu9SHDNXzKd9qoOFE1_UI1odHliHPn5pgUntv-Vtc_bRg"
      />
      <img
        alt=""
        src="https://www.collegegridirons.com/wp-content/uploads/2017/03/hrs.jpg"
      />
      <ul>
        <li>
          Stadium: Hard Rock Stadium
        </li>
        <li>
          Capacity: 65326
        </li>
      </ul>
    </div>
    <div
      class="Logos"
    >
      <h2
        id="Arizona Cardinals"
      >
        Arizona Cardinals
      </h2>
      <img
        alt=""
        src="https://lh3.googleusercontent.com/proxy/K_-WNZYkF6KiopB6FtZrdN6W5rP_4u7KBIoInwiAfPbnx_nR3KtwJ_T1rrr8c6-yagnBSlWst7ZdxHyeq4D5-qHvz5VsvSBJUH6ONecKCZMe1m0LDWtWw598sLAWTBR-hko"
      />
      <img
        alt=""
        src="https://populous.com/wp-content/uploads/2018/02/POP_00_0796_00_N25_medium.jpg"
      />
      <ul>
        <li>
          Stadium: Statefarm Stadium
        </li>
        <li>
          Capacity: 63400
        </li>
      </ul>
    </div>
    <div
      class="Logos"
    >
      <h2
        id="Jacksonville Jaguars"
      >
        Jacksonville Jaguars
      </h2>
      <img
        alt=""
        src="https://lh3.googleusercontent.com/proxy/NOhd8cahhw0xpf8vDi8vf0yNNm0EJuCRbEIxK3u8gz-00uz8BAFAm01Tbt_UAliQPrmx0VhmCO3lRoZnaVJVWMKp6Sf2tbc1WCW1plI59jDxLLo9VDrA0ptoPJBLaSueU9J1LQs"
      />
      <img
        alt=""
        src="https://downtownjacksonville.org/wp-content/uploads/2020/08/tiaa-bank-field.jpg"
      />
      <ul>
        <li>
          Stadium: TIAA Bank Field
        </li>
        <li>
          Capacity: 67164
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
